### PR TITLE
Update instrumented jars for Android 12

### PR DESCRIPTION
**Pull request by Bencodes**
The new jars got pushed up https://repo1.maven.org/maven2/org/robolectric/android-all-instrumented/

### CODE REVIEW

**Comment by utzcoz** [comment]
Hi @hoisie @brettchabot , the Robolectric 4.7 has been released, what about merging this, updating  documentation, and releasing robolectric-bazel for Robolectric 4.7?

**Comment by Bencodes** [comment]
@nkoroste want to give this a once over before I merge?

File: bazel/BUILD [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-    "@org_robolectric_android_all_instrumented_7_1_0_r7_robolectric_r1_i1//jar",
-    "@org_robolectric_android_all_instrumented_7_0_0_r1_robolectric_r1_i1//jar",
-    "@org_robolectric_android_all_instrumented_6_0_1_r3_robolectric_r1_i1//jar",
-    "@org_robolectric_android_all_instrumented_5_1_1_r9_robolectric_r2_i1//jar",
-    "@org_robolectric_android_all_instrumented_5_0_2_r3_robolectric_r0_i1//jar",
-    "@org_robolectric_android_all_instrumented_4_4_r1_robolectric_r2_i1//jar",
-    "@org_robolectric_android_all_instrumented_4_3_r2_robolectric_r1_i1//jar",
-    "@org_robolectric_android_all_instrumented_4_2_2_r1_2_robolectric_r1_i1//jar",
-    "@org_robolectric_android_all_instrumented_4_1_2_r1_robolectric_r1_i1//jar",
+    "@org_robolectric_android_all_instrumented_12_robolectric_7732740_i2//jar",
```
Comment (nkoroste) [inline bazel/BUILD:2]: Don't we want to bump the robolectric version in the README file to match these jars?  I assume every version of robolectric will have a corresponding version of the jars and we want to be in sync most of the time

Comment (Bencodes) [reply bazel/BUILD:2]: Updated

**Review by nkoroste — APPROVED** [review]

**Pull request merged**